setup-wizard.yml

```
@@ -3,10 +3,10 @@ fields:
   - title: Execution Client Config Mode
     id: config_mode
```

```suggestion
    title: Execution Client Selection Configuration Mode
```
again keeping with the preexisting style